### PR TITLE
Assignment 8 - Priyaa

### PR DESCRIPTION
**Pull request by PriyaaIyer**
Changes:
- Page Title added (all assignments)
- Changed \<b> to \<strong> 
- Added \<em>
- Added metadata
- Used ordered lists
- Used <thead> and <tbody> tags 

<a href="https://gitpod.io/#https://github.com/aravindvnair99/ACM-Amrita-2019-2020-Assignments/pull/54"></a>

### CODE REVIEW

File: Priyaa/HTML Basics Remake/Assignment-1.html [hunk 1]
```diff
@@ -1,49 +1,64 @@
...
-<p><em>From Wikipedia, the free encyclopedia</em></p>
+   <h1>Merlo J. Pusey</h1>
+   <hr>
+   <p><cite>From Wikipedia, the free encyclopedia</cite></p>
+
+   <img src="https://i.ebayimg.com/images/g/X5wAAOSwARZXoO9s/s-l300.jpg" width="250" , height="200" />
 
-<img src="https://i.ebayimg.com/images/g/X5wAAOSwARZXoO9s/s-l300.jpg" width="250" , height="200" />
+   <p><strong>Merlo John Pusey</strong>(February 3, 1902, in Woodruff, Utah – November 22, 1985, in Washington,
+      D.C.)[1][2] was an <u>American biographer</u> and <u>editorial writer</u>. He won the 1952 Pulitzer Prize for
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-1.html]:
```suggestion
      D.C.)<sup>[1][2]</sup> was an <u>American biographer</u> and <u>editorial writer</u>. He won the 1952 Pulitzer Prize for
```

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 2]
```diff
@@ -2,90 +2,65 @@
...
+        <meta name='viewport' content='width=device-width, initial-scale=1'>
+
 </head>
 
 <body>
 
 
 
-    <h1><b>Register</b></h1>
+        <h1><b>Register</b></h1>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: No need of `b` here, `h1` should be good enough.

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 3]
```diff
@@ -2,90 +2,65 @@
...
 
-    <form id="registration_form" method="post">
-        <label for="first name"><b>First name:</b></label>
-        <input type="text" 
-                name="firstname" 
-                placeholder="First Name" 
-                autocomplete="given-name" 
-        required>
+                <label for="last name"><strong>Last name:</strong></label>
+                <input type="text" name="lastname" placeholder="Last Name" autocomplete="family-name" required>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: last name 'id' is missing

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 4]
```diff
@@ -2,90 +2,65 @@
...
-        <label for="Password"><b>Password</b></label>
-        <input type="password" 
-                name="password" 
-                placeholder="Password" 
-                minlength="5" 
-                maxlength="15" 
-                autocomplete="new-password" 
-        required>
+                <label for="Birthday"><strong>Birthday</strong></label>
+                <input type="date" name="birthday" autocomplete="bday" required>
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: Birthday `id` is missing

File: Priyaa/HTML Basics Remake/Assignment-3.html [hunk 5]
```diff
@@ -2,90 +2,65 @@
...
 
-        <label for="Birthday"><b>Birthday</b></label>
-        <input type="date" 
-                name="birthday" 
-                autocomplete="bday" 
-                required>
-        <br><br>
 
 
+                <label for="terms and conditions"><strong>I agree to the terms and conditions</strong></label><input
```
Comment (VinayakTekade) [inline Priyaa/HTML Basics Remake/Assignment-3.html]: terms and conditions `id` is missing

**Review by VinayakTekade — CHANGES_REQUESTED** [review]

**Comment by VinayakTekade** [comment]
Nice work. Just these few changes and you are good to go.

**Comment by PriyaaIyer** [comment]
Made necessary changes, formatted code and pushed it.
Unable to create a new pull request.

**Review by VinayakTekade — DISMISSED** [review]

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
- Remove unnecessary spaces.
- Set to tab indentation of 4. You can confirm by looking in VS Code:
image
- Then format document.

**Comment by aravindvnair99** [comment]
@PriyaaIyer I've updated your PR description. Use proper GFM syntax henceforth with proper escape characters. Refer this: [GFM - GitHub Guides](https://guides.github.com/features/mastering-markdown/)

**Comment by PriyaaIyer** [comment]
Removed unnecessary spaces
Set indentation using tab to 4
Formatted document

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@PriyaaIyer Make the changes intimated to you on the group.

**Pull request closed without merging**